products/serializers.py

```diff
@@ -16,6 +18,11 @@ def create(self, validated_data):
         # pylint: disable=no-member
         return Product.objects.create(**validated_data)
 
+    def update(self, instance, validated_data):
+        os.remove(instance.image.path)
```

Comment:
I believe this can throw an exception and we should catch it.

Reply:
This line is not needed anymore. I have removed it.